index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <nav class="container-logo">
+            <a href="#">
+                <img src="images/logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img src="images/Union.png" alt="union" class="union">
+            </a>
+        </nav>
+        
+        <div class="main-container">
+            <div class="main-text">
+                <h2>I’m MeeMaw <br> Glad to see you!</h2>
```

Comment:
- [ ] Kindly use an `<h1>` tag for the main title instead  of a `<h2>` tag.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.